assets/scss/sidebar-widgets.scss

```diff
@@ -70,7 +96,7 @@
 
     //subpages nav alteration
      ul > li {
-      border-bottom: 1px solid #d8dde0;
+      // border-bottom: 1px solid #d8dde0;
```

Comment:
can this be left in please? Some of our menus get very busy, and this helps delineate when there are sub pages. Also, the left hand margin on subpages has grown to 1.5rem (I think with the removal of the .nhsuk-list-panel__list extension on line 22 above), and this will make sub-subs very squished. Can it be reduced down to 0.5rem please.